src/main/java/com/elixr/poc/rest/response/PurchaseResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.elixr.poc.rest.response;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This Class holds the response attributes for purchase related end points
+ */
+
+@Data
+public class PurchaseResponse extends ErrorResponse {
+
+    @Id
+    private UUID id;
+    private String userName;
+    private String product;
+    private String amount;
+    private String date;
+
+    @Builder(builderMethodName = "purchaseBuilder")
```

Comment:
Use Lombok annotation for all argument constructor

Reply:
updated